### PR TITLE
qa: fix proc exit status check

### PR DESCRIPTION
**Pull request by batrick**
Fixes: f30a84b6a7a21670d8b5c5a7fe19df70e4371260
Fixes: https://tracker.ceph.com/issues/47677
Signed-off-by: Patrick Donnelly <pdonnell@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by joscollin — APPROVED** [review]

File: qa/tasks/ceph.py [hunk 1]
```diff
@@ -177,12 +177,12 @@ def invoke_logrotate(self):
             while not self.stop_event.is_set():
                 self.stop_event.wait(timeout=30)
                 try:
-                    p = ctx.cluster.run(
-                        args=['sudo', 'logrotate', '/etc/logrotate.d/ceph-test.conf'],
-                        wait=False,
-                        stderr=StringIO()
+                    procs = ctx.cluster.run(
```
Comment (tchaikov) [inline qa/tasks/ceph.py:180]: nit, s/procs/proc/

Comment (batrick) [reply qa/tasks/ceph.py:180]: It's actually a list of procs returned by `ctx.cluster.run` so I thought this name was appropriate.

**Comment by batrick** [comment]
I'm waiting for an instance of this failure to show up in my testing and will merge once the fix is confirmed to work.

**Comment by batrick** [comment]
At last, I was able to verify the fix:

````
2020-10-10T01:11:01.639 INFO:teuthology.orchestra.run.smithi053:> sudo logrotate /etc/logrotate.d/ceph-test.conf
2020-10-10T01:11:01.641 INFO:teuthology.orchestra.run.smithi165:> sudo logrotate /etc/logrotate.d/ceph-test.conf
2020-10-10T01:11:01.644 INFO:teuthology.orchestra.run.smithi193:> sudo logrotate /etc/logrotate.d/ceph-test.conf
2020-10-10T01:11:01.717 INFO:teuthology.orchestra.run.smithi193.stderr:error: error renaming temp state file /var/lib/logrotate/logrotate.status.tmp
2020-10-10T01:11:01.719 DEBUG:teuthology.orchestra.run:got remote process result: 1
2020-10-10T01:11:01.719 INFO:tasks.ceph:ignoring transient state error: Command failed on smithi193 with status 1: 'sudo logrotate /etc/logrotate.d/ceph-test.conf'
````

From: /ceph/teuthology-archive/pdonnell-2020-10-09_05:10:50-multimds-wip-pdonnell-testing-20201009.020122-distro-basic-smithi/5509020/teuthology.log

**Pull request merged**